### PR TITLE
Show message for Youtube video [stage-3]

### PR DESCRIPTION
**Pull request by donnapep**
In preview, JWPlayer throws the error "Error loading Youtube: Video could not be played."

In player, JWPlayer throws the error "Error loading player: No playable sources found" and the URL must be checked to see if it's a Youtube video.

### CODE REVIEW

**Comment by donnapep** [comment]
@rodrigopavezi @stulees Please review. Thx.

**Comment by rodrigopavezi** [comment]
@donnapep looks good. :+1:

File: src/widget/video.js [hunk 1]
```diff
@@ -285,7 +292,13 @@ RiseVision.Video = (function (gadgets) {
   function playerError(error) {
     var details = null,
       params = {},
-      message = "Sorry, there was a problem playing the video.";
+      url = "",
+      message = "Sorry, there was a problem playing the video.",
+      MEDIA_ERROR = "Error loading media: File could not be played",
```
Comment (stulees) [inline src/widget/video.js]: @donnapep Presumably these are capitalized to act as constants, right? If so I think they're better suited to be assigned in this `video.js` module vars as opposed to only in this function.

Comment (donnapep) [inline src/widget/video.js]: @stulees I tend to disagree on the premise that variables should use the minimum scope required to do their job.

Comment (stulees) [inline src/widget/video.js]: @donnapep Agreed, but capitalizing these vars gives the indication they're acting as constants, which I don't see much benefit for using constants within one function of a module. Not going to hold up the PR on it though, thanks.

**Comment by stulees** [comment]
@donnapep :+1:

File: src/widget/video.js (hunk 1 above)
Comment (donnapep) [inline src/widget/video.js]: @stulees OK, will ponder this a bit. I originally had them as vars and then thought constants would be a more clear indicator that the values shouldn't be changed. Thx.

**Pull request merged**